ledger/store/accountsV2.go

```diff
@@ -84,6 +86,81 @@ func (r *accountsV2Reader) AccountsTotals(ctx context.Context, catchpointStaging
 	return
 }
 
+// AccountsAllTest iterates the account table and returns a map of the data
+// It is meant only for testing purposes - it is heavy and has no production use case.
+func (r *accountsV2Reader) AccountsAllTest() (bals map[basics.Address]basics.AccountData, err error) {
```

Comment:
nit: since this is part of an interface probably need a better name AccountsAllTest -> GetAllAccountsTesting or similar